lib/schemas/observation.gql

```diff
@@ -73,16 +73,14 @@ type EconomicEvent {
   inScopeOf: [AnyType!]
 
   "Reference to an agreement between agents which specifies the rules or policies or calculations which govern this economic event."
-  agreedIn: AnyAgreement
+  agreedIn: URI
```

Comment:
`AnyAgreement` was a union between a struct which holds an external `URI`, and one of VF's own `Agreement` records. I thought it would be sensible to provide for both? Surely we would want to be able to reference an `Agreement` record in this way as well as something external?

Reply:
I'm trying to separate them.  Perhaps I am thinking too narrowly based on our experience.  But here is my thinking:  First of all, Agreement is too broadly named, but I didn't see I could get that changed for this round, wanted to re-discuss with @elf-pavlik , although maybe I need to give that hope up.  It is really Exchange or ExchangeAgreement, only used for exchanges, and other kinds of agreements, which can be very complex, would be outside VF.  So I don't see any time VF Agreements would be used as a reference to the "reason" some calculation was done as it was in an event or whatever.  Even if so, that would be already referenced using clauseOf or realizationOf.

Reply:
I guess with that I wonder why we would have `ExchangeAgreement` in VF at all, and not just have agreements for exchanges managed externally, same as agreements of any other kind?

Anyway this is fine for this change then, will leave as it is and also remove the `AnyAgreement` type.

Reply:
> I wonder if the word "agreement" is causing semantic problems?

That sounds like it could be a thing (;